src/components/AudioPlayer.vue

```diff
@@ -95,13 +130,22 @@ export default {
   beforeDestroy() {
     this.audioPlayer.close();
   },
+  data() {
+    return {
+      // isSmallPlayer: true,
+      playerHeight: 100,
```

Comment:
The value of this is only ever `100`. Is this intended behaviour?

Reply:
This is for later usage.